### PR TITLE
Fix a wrong link in partitioned-table.md

### PR DESCRIPTION
**Pull request by dcswinner01**
### What is changed, added or deleted? (Required)
Fix a wrong link in partitioned-table.md.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by dcswinner01** [comment]
https://docs.pingcap.com/zh/tidb/v6.1/partitioned-table  文档中关于[TiFlash](https://docs.pingcap.com/zh/tidb/stable/tikv-overview)：TiFlash Replica 数量相同。的超链接错误，链接到 TiKV 了。

File: partitioned-table.md [hunk 1]
```diff
@@ -720,7 +720,7 @@ Empty set (0.00 sec)
 请注意对于以下 TiDB 专有的特性，当表结构中包含这些特性时，在 TiDB 中使用 `EXCHANGE PARTITION` 功能不仅需要满足 [MySQL 的 EXCHANGE PARTITION 条件](https://dev.mysql.com/doc/refman/8.0/en/partitioning-management-exchange.html)，还要保证这些专有特性对于分区表和非分区表的定义相同。
 
 * [Placement Rules in SQL](/placement-rules-in-sql.md)：Placement Policy 定义相同。
-* [TiFlash](/tikv-overview.md)：TiFlash Replica 数量相同。
+* [TiFlash](/tiflash-overview.md)：TiFlash Replica 数量相同。
```
Comment (TomShawn) [inline partitioned-table.md:723]:
```suggestion
* [TiFlash](/tiflash/tiflash-overview.md)：TiFlash Replica 数量相同。
```

**Review by TomShawn — APPROVED** [review]
谢谢指出，LGTM👍

**Pull request merged**